recipes/rapidcheck/all/conanfile.py

```diff
@@ -20,11 +20,21 @@ class RapidcheckConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "enable_rtti": [True, False],
+        "enable_catch": [True, False],
+        "enable_gmock": [True, False],
+        "enable_gtest": [True, False],
+        "enable_boost": [True, False],
+        "enable_boost_test": [True, False]
```

Comment:
Could this be one option?

```suggestion
        "enable_framework": [False, "catch", "gmock", "gtest", "boost", "boost_test"],
```

Reply:
No I don't think so as you can enable multiple integrations at the same time.